Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py

```diff
@@ -0,0 +1,395 @@
+                         skip_token: str = None) -> dict:
+        """
+        List risky users.
+
+        Args:
+            risk_state (str): Risk State to retrieve.
+            risk_level (str): Specify to get only results with the same Risk Level.
+            limit (int): Limit of results to retrieve.
+            skip_token (str): Skip token.
+
+        Returns:
+            response (dict): API response from AzureRiskyUsers.
+        """
+        params = remove_empty_elements({'$top': limit,
+                                        '$skiptoken': skip_token,
+                                        '$filter': build_query_filter(risk_state, risk_level)})
+
+        return self.ms_client.http_request(method='GET',
+                                           url_suffix="identityProtection/riskyUsers",
+                                           params=params)
+
+    def risky_user_get(self, id: str) -> dict:
+        """
+        Get risky user by ID.
+
+        Args:
+            id (str): Risky user ID to get.
+
+        return:
+            Response (dict): API response from AzureRiskyUsers.
+        """
+        return self.ms_client.http_request(method='GET',
+                                           url_suffix=f'identityProtection/riskyUsers/{id}')
+
+    def risk_detections_list(self, risk_state: Optional[str], risk_level: Optional[str], limit: int,
+                             skip_token: str = None) -> dict:
+        """
+        Get a list of the Risk Detection objects and their properties.
+
+        Args:
+            risk_state (str): Risk State to retrieve.
+            risk_level (str): Specify to get only results with the same Risk Level.
+            limit (int): Limit of results to retrieve.
+            skip_token (int): Skip token.
+
+        return:
+            Response (dict): API response from AzureRiskyUsers.
+        """
+        params = remove_empty_elements({'$top': limit,
+                                        '$skiptoken': skip_token,
+                                        '$filter': build_query_filter(risk_state, risk_level)})
+
+        return self.ms_client.http_request(method='GET',
+                                           url_suffix="/identityProtection/riskDetections",
+                                           params=params)
+
+    def risk_detection_get(self, id: str) -> dict:
+        """
+        Read the properties and relationships of a riskDetection object.
+
+        Args:
+            id (str): ID of risk detection to retrieve.
+
+        Return:
+            Response (dict): API response from AzureRiskyUsers.
+        """
+        return self.ms_client.http_request(method='GET',
+                                           url_suffix=f'/identityProtection/riskDetections/{id}')
+
+
+def build_query_filter(risk_state: Optional[str], risk_level: Optional[str]) -> Optional[str]:
+    """
+    Build query filter for API call, in order to get filtered results.
```

Comment:
Let's add a link to the syntax in the API documentation here

Reply:
Done